src/middle/Stan_math_signatures.ml

```diff
@@ -849,7 +849,6 @@ let () =
   List.iter
     ~f:(fun x -> add_unqualified ("add", ReturnType x, [x; x], AoS))
     bare_types ;
-  add_unqualified ("add", ReturnType UComplex, [UComplex; UReal], AoS) ;
```

Comment:
We are removing these due to the automatic conversion issue? Same Q for norm.

Reply:
These are related to #998. I think that they were added because the PR misunderstood which conversions should be automatic.

Norm is being removed for a different reason: g++ crashes at the moment if you actually call it with a real input. 

Rewriting the complex tests is part of the PRs related to the above. I can add it separately here if you'd like, but part of the testing is testing it on real/int inputs which wouldn't work yet

Reply:
Ok, I am good with that.